<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WsdlMethodView</class>
 <widget class="QWidget" name="WsdlMethodView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WsdlMethodView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="WsdlTreeViewHeaderWidget" name="TreeHeader" native="true"/>
       </item>
       <item>
        <widget class="QTreeView" name="TreeView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="WsdlXmlViewHeaderWidget" name="XmlHeader" native="true"/>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="XmlView">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WsdlTreeViewHeaderWidget</class>
   <extends>QWidget</extends>
   <header>WsdlTreeViewHeaderWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WsdlXmlViewHeaderWidget</class>
   <extends>QWidget</extends>
   <header>WsdlXmlViewHeaderWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
